### ISSUE TITLE
Icon buttons don't have labels

### ISSUE THREAD

**Issue by juuso-vaadin** [vaadin-component-factory/vcf-pdf-viewer#14]
Icon buttons in the toolbar should have either tooltips or aria-labels describing the purpose of each action.

Using Tooltip introduced in V23.3 will help both regular users and screen reader users.
For earlier Vaadin versions adding `aria-label` will solve the screen reader accessibility issue.

**Comment by paodb** [comment]
Closed by #16
